public/app/core/utils/fetch.ts

```diff
@@ -7,19 +7,20 @@ export const parseInitFromOptions = (options: BackendSrvRequest): RequestInit =>
   const isAppJson = isContentTypeApplicationJson(headers);
   const body = parseBody(options, isAppJson);
 
-  if (options?.withCredentials) {
+  if (options?.credentials) {
     return {
       method,
       headers,
       body,
-      credentials: 'include',
+      credentials: options.credentials,
     };
   }
 
   return {
     method,
     headers,
     body,
+    credentials: 'same-origin',
```

Comment:
can you describe how this impacts https://github.com/grafana/grafana/issues/27007 ? Are you using multiple domains?

Reply:
We use some android browsers running in TV device for visualizing grafana dashboards. These old version browsers do not use `same-origin` as the default value. So after input username and password in the login page, the cookie `grafana_session` is not set correctly and then the browser will redirect to the login page again.